src/Modules/Unix/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1

```diff
@@ -9,14 +9,16 @@ PowerShellVersion="3.0"
 NestedModules="Microsoft.PowerShell.Commands.Management.dll"
 HelpInfoURI = 'https://go.microsoft.com/fwlink/?linkid=855958'
 FunctionsToExport = @()
-AliasesToExport = @("gtz")
+AliasesToExport = @("gcb", "gtz", "scb")
```

Comment:
It seems our current intention is to delegate aliases to users. So I'd remove this.

Reply:
Those aliases already exist in Windows PowerShell so users would expect them.